install/resources/kafka/cluster-example-metrics.configmap.yaml

```diff
@@ -0,0 +1,183 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: cluster-example-metrics
```

Comment:
I wonder if we should use _examples_ in the names? I'm assuming this should not be changed.

Reply:
I may not be following, do you mean it should be called `cluster-examples-metrics`?

Reply:
Not clear! I was thinking something like `cluster-metrics`, as I'm assuming these metrics configurations shouldn't be adjusted. Using _example_ in the name sounds like this is only one way of configuring the metrics

Reply:
I agree. The naming of this and other resources in the same directory are all like `cluster-example` to align with the name of the Kafka CR. Maybe we can change them all to `demo-cluster` or `trial-cluster`? I'm fine with other suggestions too.

Reply:
If we uses `demo-cluster-metrics`, I wonder if we're in danger of making the name obsolete when it moves beyond demo?  The only other name I can think of is `console-kafka-metrics`, as we're scarping kafka metrics for the console.

Reply:
I think the Kafka resources will always be for demonstration purposes to some extent. Let's go with `console-kafka` rather than `demo-cluster` for these resources.